lib/diaspora_federation/discovery/web_finger.rb

```diff
@@ -146,15 +165,38 @@ def to_s
         end
       end
 
+      def add_aliases_to(doc)
```

Comment:
Maybe we should use `doc` as a method instead of passing it as a parameter?
```ruby
def doc
  @doc ||= XrdDocument.new
end
```

It doesn't make much difference, but using a method instead of parameter passing feels like a better design.